webcomponents-loader.js

```diff
@@ -45,10 +45,24 @@
     document.head.appendChild(newScript);
   } else {
     // Ensure `WebComponentsReady` is fired also when there are no polyfills loaded.
-    requestAnimationFrame(function() {
-      // Reset the HTMLImports stub.
-      window['HTMLImports'] = null;
-      window.dispatchEvent(new CustomEvent('WebComponentsReady'));
-    });
+    // however, we have to wait for the document to be in 'interactive' state,
+    // otherwise a rAF may fire before scripts in <body>
+
+    var fire = function() {
```

Comment:
This logic is required also in https://github.com/webcomponents/webcomponentsjs/blob/fix-wcr/entrypoints/webcomponents-sd-ce-index.js#L21